test/utilities/distributions.jl

```diff
@@ -0,0 +1,114 @@
+using Turing
```

Comment:
Looks like somehow tab === 8-spaces in this file. Could we please sort out the formatting?

Reply:
I think I've fixed this, but could you confirm?